src/xtgeo/cube/cube1.py

```diff
@@ -729,6 +728,11 @@ def from_file(self, sfile, fformat="guess", engine="segyio"):
         fobj = xtgeosys._XTGeoFile(sfile)
         fobj.check_file(raiseerror=OSError)
 
+        if engine is not None:
+            logger.warning("'engine' is a depracated option.")
```

Comment:
Use warning?
```suggestion
warnings.warn("engine is a deprecated option", DeprecationWarning)
```

Reply:
Yes, maybe. Currently, xtgeo displays warnings in many different ways. Perhaps  we should have only one way of displaying warnings? I do think using warnings instead of the logger is the right choice, but perhaps we should go through and unify?

Reply:
Changed it to `logger.warndeprecated` which does the same thing underneath the hood.